core/themes/seven/css/style.css

```diff
@@ -645,33 +645,36 @@ tr.selected td {
  *   to apply all padding to the inner .fieldset-wrapper instead.
  */
 fieldset {
-  position: relative;
   max-width: 100%;
   margin: 1em 0;
-  padding: 3em 0 0 0;
-  border-radius: 4px;
+  padding: 0;
   background-color: #fff;
-  border: 2px solid #EAEAEA;
+  border: none;
   min-width: 0;
 }
 fieldset .fieldset-legend {
-  position: absolute;
-  left: 0; /* LTR */
-  top: 0;
-  width: 100%;
-  margin-top: .25em;
-  padding-left: 1em; /* LTR */
+  display: block;
   text-transform: uppercase;
   box-sizing: border-box;
+  border-radius: 4px;
+  border: 2px solid #EAEAEA;
+  background-color: #fff;
 }
-[dir="rtl"] fieldset .fieldset-legend {
-  right: 0;
-  padding-right: 1em;
-  padding-left: 0;
+fieldset:not(.collapsible) .fieldset-legend {
+  padding: .3em .3em .3em 1em; /* LTR */
+}
+[dir="rtl"] fieldset:not(.collapsible) .fieldset-legend {
+  padding: .3em 1em .3em .3em;
+}
+fieldset:not(.collapsible) .fieldset-legend,
+fieldset.collapsible:not(.collapsed) .fieldset-legend {
+  border-bottom: none;
```

Comment:
This line removes the border between the fieldset legend and the fieldset content, to retain the current style:

![screencapture-3930-lndo-site-444-admin-appearance-settings-basis-2020-05-19-01_27_46](https://user-images.githubusercontent.com/2423362/82266046-42d79400-9971-11ea-8248-953a4d65217f.png)

If most people find this border acceptable/preferable, then we can remove this bit and have the border added.